offix/src/main/java/org/aerogear/offix/interceptor/OffixConflictCallback.kt

```diff
@@ -0,0 +1,57 @@
+package org.aerogear.offix.interceptor
+
+import android.util.Log
+import com.apollographql.apollo.api.Error
+import com.apollographql.apollo.exception.ApolloException
+import com.apollographql.apollo.interceptor.ApolloInterceptor
+import org.aerogear.offix.*
+import org.json.JSONObject
+import com.google.gson.Gson
+
+/*
+OffixConflictCallback
+ */
+class OffixConflictCallback : ApolloInterceptor.CallBack {
+    private val TAG = javaClass.simpleName
+
+    override fun onResponse(response: ApolloInterceptor.InterceptorResponse) {
+
+        /* Check if the conflict is present in the response of not using the ConflictResolutionHandler class.
+         */
+        if (ConflictResolutionHandler().conflictPresent(response.parsedResponse)) {
+
+            /* Parse the response into a json object from the server and extract the serverState and clientState.
+               Make an object of ServerClientData and add to the list.
+            */
+            val extensions = JSONObject(
```

Comment:
Don't make a JSONObject here, simply get('clientData') and get('serverData') from here and convert it to a hashMap, WDYT?